.circleci/config.yml

```diff
@@ -57,6 +57,19 @@ jobs:
       - persist_to_workspace:
           root: ~/repo
           paths: .
+  slither:
+      docker:
+          - image: trailofbits/eth-security-toolbox
+      steps:
+          - checkout
+          - attach_workspace:
+              at: .
+          - run:
+              command: |
+                  set +e
+                  slither .
+                  exit 0
+              name: Show Slither output
```

Comment:
I think this by itself is not enough, you need to add it below to the `workflows.ci.jobs` section, to actually get it to be executed in the pipeline.

Reply:
thanks! I think I fixed it: https://app.circleci.com/pipelines/github/nayms/contracts/1125/workflows/9738aed8-6e8f-446d-84f4-358b8617898c/jobs/1735/parallel-runs/0/steps/0-103